File: HD/hw2/Sqrts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Sqrts {
  public static void main(String args[]) {
    System.out.println("n\tsqrt(n)");
    System.out.println("----------------");
    int n;
    for (n=0;  n<=5;  n++)
      System.out.println(n + "\t" + Math.sqrt(n));
    return;

  }
}

File: HD/hw2/sqrts.c


/* print square roots in C language.  R. Brown, 9/2010 */

#include <stdio.h>
#include <math.h>

main()
{
  int n;
  printf("n\tsqrt(n)\n----------------\n");
  for (n=0;  n<=5;  n++)
    printf("%d\t%f\n", n, sqrt(n));
  return 0;
}

